curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3477aefa51bfc29327200b.md

```diff
@@ -15,6 +15,14 @@ Start by rewriting the styles you have created into the `styles.css` file. Make
 
 # --hints--
 
+You should only have one selector.
+
+```js
+(getUserInput) => {
+  assert(getUserInput('editableContents').match(/text-align:\s*center;?/)?.length === 1);
+}
+```
+
```

Comment:
```suggestion
```

I know this is what was discussed in the issue, but I'm not sure I like this test here - it really doesn't make sense. It says, "You should only have one selector." - but it tests for the CSS property and value. Users could add a selector without the property and get this as the hint. We could maybe just delete this test? or maybe move it below the next one or the one after that - which would ensure they have the selector first. 

Also, the test doesn't really work - the regex doesn't have the `g` flag so even if a user adds the `text-align: center` more than once, it will only match the first one - and pass.

I don't really see a need for this test, I vote to delete it - but I'm open to suggestions.

Reply:
Uh, you are right, it's a pretty crappy test...